src/store/user/index.js

```diff
@@ -73,6 +73,8 @@ const mutations = {
       id: null,
       email: null,
       username: null,
+      first_name: '',
+      last_name: '',
```

Comment:
These were missing for long enough that I think we should add a templating function for this user state. We can do that in a follow-up PR though.